src/result_generation/height/height_pose3d.py

```diff
@@ -0,0 +1,139 @@
+import uuid
+from datetime import datetime
+
+import cv2
+import numpy as np
+from bunch import Bunch
+import log
+from cgmml.common.depthmap_toolkit.depthmap import Depthmap
+from cgmml.models.HRNET.hrnet3d import (convert_2dskeleton_to_3d,
+                                        write_skeleton_into_obj)
+from joblib import load
+from result_generation.height.height import HeightFlow
+from result_generation.pose_prediction.inference import (inference_artifact,
+                                                         init_pose_prediction)
+from utils.config_train import CONFIG_TRAIN
+from utils.pose_utils import get_features_from_fpath
+
+logger = log.setup_custom_logger(__name__)
+
+
+class HeightFlowPose3D(HeightFlow):
+    def run_flow(self):
+        if self.scan_version in ["v0.9", "v1.0.2"] and self.scan_type in [200, 201, 202]:
+            start_time = datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ')
+            mean_prediction = self.pose_prediction_artifacts()
+            generated_timestamp = datetime.now().strftime('%Y-%m-%dT%H:%M:%SZ')
+            self.post_height_results(mean_prediction, generated_timestamp, start_time)
+
+    def pose_prediction_artifacts(self):
+        """Blur the list of artifacts"""
+        pose_prediction = init_pose_prediction()
+        MODEL = load('models/pose-3d/2021q4-points3d-rf-height-28k-200and201.joblib')
+        mean_prediction = 0
+        processed_artifacts = 0
+        for image_artifact, artifact in zip(self.image_artifacts, self.artifacts):
+            input_rgb_path = self.result_generation.get_input_path(self.scan_rgb_directory, image_artifact['file'])
+            logger.info("%s %s", "input_path of image to perform Pose prediction:", input_rgb_path)
+            no_of_body_pose, _, _, self.persons_coordinates = inference_artifact(
+                pose_prediction, input_rgb_path, self.scan_type)
+            if no_of_body_pose == 1:
+                input_depth_path = self.result_generation.get_input_path(self.scan_directory, artifact['file'])
+                self.dmap = Depthmap.create_from_zip_absolute(
```

Comment:
Using default calibration file for all scans.
Calibration file is uploaded from device for each scan, should we use that instead?